### PR TITLE
Show Labels Sidebar on Annotation Page #166

### PR DESCRIPTION
**Pull request by arinkulshi-skylight**
# Pull Request

## Description
https://github.com/CDCgov/IDWA/issues/166

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: OCR/frontend/src/pages/AnnotateTemplate.tsx [hunk 1]
```diff
@@ -34,44 +47,123 @@ const AnnotateTemplate: React.FC = () => {
...
+        <li
+          key={item.name}
+          style={{
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "center",
+            padding: "8px 0",
+          }}
+        >
+          <div style={{ display: "flex", alignItems: "center" }}>
```
Comment (knguyenrise8) [inline OCR/frontend/src/pages/AnnotateTemplate.tsx:76]: can we use the [style guide](https://designsystem.digital.gov/utilities/display/) for all of the class names

Comment (arinkulshi-skylight) [reply OCR/frontend/src/pages/AnnotateTemplate.tsx]: I have tried to follow the style guide as close as I can please let me know if I missed anything! Happy to pair on it as well.

**Review by knguyenrise8 — APPROVED** [review]

**Review by knguyenrise8 — APPROVED** [review]

**Review by knguyenrise8 — APPROVED** [review]
